guides/common/modules/proc_recovering-corrupted-content-on-a-smart-proxy.adoc

```diff
@@ -0,0 +1,34 @@
+[id="Recovering_Corrupted_Content_on-a-Smart-Proxy_{context}"]
+= Recovering corrupted content on a {SmartProxy}
+
+In case of content corruption on a {SmartProxy}, you can recover it by using the `verify-checksum` command in Hammer CLI.
+The `verify-checksum` command works for optional parameters associated with the {SmartProxy} such as `content-view id`, `environment-id`, or `repository-id` to limit the repair.
+You can run the `verify-checksum` command without any filters so it runs on all content on the {SmartProxy}.
```

Comment:
Instead of describing the parameters, try saying that they can repair content in the given resource (content view, lifecycle environment, repository) or all content on the SmartProxy. Let's see if you can make the two lines into a single sentence, impress me :)